### ISSUE TITLE
Fix broken event type decoding against master edgeware-node (3.0.8)

### ISSUE THREAD

**Issue by drewstone** [hicommonwealth/chain-events#21]
Ever so often running the listener against `edgeware-local` I see the following:
```
Unable to decode storage system.events: createType(Vec<EventRecord>):: findMetaEvent: Unable to find Event with index 0xd863/[216,99]
2020-09-05 23:27:54        RPC-CORE: getStorage(key: StorageKey, at?: BlockHash): StorageData:: createType(Vec<EventRecord>):: findMetaEvent: Unable to find Event with index 0xd863/[216,99]
(node:48585) UnhandledPromiseRejectionWarning: Error: createType(Vec<EventRecord>):: findMetaEvent: Unable to find Event with index 0xd863/[216,99]
    at createTypeUnsafe (/Users/drewstone/code/commonwealth/chain-events/node_modules/@polkadot/types/create/createType.js:69:11)
    at Rpc._formatStorageData (/Users/drewstone/code/commonwealth/chain-events/node_modules/@polkadot/rpc-core/index.js:394:40)
    at Rpc._formatOutput (/Users/drewstone/code/commonwealth/chain-events/node_modules/@polkadot/rpc-core/index.js:360:21)
    at callWithRegistry (/Users/drewstone/code/commonwealth/chain-events/node_modules/@polkadot/rpc-core/index.js:222:62)
    at processTicksAndRejections (internal/process/task_queues.js:89:5)
```

**Comment by jnaviask** [comment]
My guess is that this has to do with type overrides at API initialization time -- how are we doing this? Depending on the substrate version, we may need these:
```
  'RewardDestination': 'RewardDestinationTo257',
  'CompactAssignments': 'CompactAssignmentsTo257',
```

**Comment by jnaviask** [comment]
Fixed in #25.
